lib/board.rb

```diff
@@ -0,0 +1,68 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells, :board, :token
+
+  turn_count = 0
+
+  def initialize
+    self.cells = Array.new(9, " ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.